### PR TITLE
ARROW-8639: [C++][Plasma] Require gflags

### PR DESCRIPTION
**Pull request by kou**

### CODE REVIEW

**Comment by kou** [comment]
@github-actions crossbow submit centos-6-amd64 centos-7-amd64 ubuntu-xenial-amd64

**Comment by emkornfield** [comment]
+1

**Comment by emkornfield** [comment]
Is centos-6-amd64 failure OK?

**Comment by kou** [comment]
It's not OK...
We should not use old gflags (< 2.1.0).

**Comment by kou** [comment]
@github-actions crossbow submit centos-6-amd64 centos-7-amd64 ubuntu-xenial-amd64

**Comment by kou** [comment]
@github-actions crossbow submit centos-*-amd64

**Review by kszucs — APPROVED** [review]
LGTM

File: cpp/cmake_modules/FindgflagsAlt.cmake [hunk 1]
```diff
@@ -15,6 +15,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
+# TODO: Support version detection.
```
Comment (kszucs) [inline cpp/cmake_modules/FindgflagsAlt.cmake:18]: Created a follow-up ticket https://issues.apache.org/jira/browse/ARROW-8653

Comment (kou) [reply cpp/cmake_modules/FindgflagsAlt.cmake:18]: Thanks!

**Pull request closed without merging**